lib/db/set.go

```diff
@@ -338,17 +338,22 @@ func (s *Snapshot) Sequence(device protocol.DeviceID) int64 {
 	return s.meta.Counts(device, 0).Sequence
 }
 
-// RemoteSequence returns the change version for the given folder, as
-// sent by remote peers. This is guaranteed to increment if the contents of
-// the remote or global folder has changed.
-func (s *Snapshot) RemoteSequence() int64 {
```

Comment:
I don't see this coming up in the diff - this was completely unused before?! Good thing, summing all remote sequences seemed a bit strange :)

Reply:
It was unused, yes.